corehq/apps/data_interfaces/migrations/0034_case_name_actions.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+from corehq.apps.data_interfaces.models import CaseDeduplicationActionDefinition
+
+
+def forwards(apps, schema_editor):
+    replace_property_value('case_name', 'name')
+
+
+def backwards(apps, schema_editor):
+    replace_property_value('name', 'case_name')
```

Comment:
yes, I mentioned this in the ticket description. Given that values that are currently `"name"` don't work correctly anyway, if we had to undo the migration, it would only cause rules that 'incorrectly' referenced `"name"` to handle updates.

Reply:
I also should add -- I'm open to alternatives, but it didn't seem worth the overhead of storing the initially affected migration records somewhere so we could undo the migration on only those records.